cmd/ctr/commands/run/run_unix.go

```diff
@@ -334,9 +335,15 @@ func NewContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 		if limit != 0 {
 			opts = append(opts, oci.WithMemoryLimit(limit))
 		}
+		var cdiDeviceIDs []string
```

Comment:
A question to the reviewers: Does CTR read the containerd config, and if so, do we want to use the same feature flag to control this behaviour here. We may also, for example, want to configure the CDI registry with the configured CDI paths -- although this is not configurable for existing CLI-based implementations.

Reply:
I think in general, it may be a good idea to instantiate an instance of the CDI registry in the CLI code path explicily instead of relying on the instantiation in the (now) OCI package using the defaults.

Reply:
> Does CTR read the containerd config

No

Reply:
Thanks for the clarification. I will update the PR to instantiate the singleton CDI registry with auto-refresh disabled and also provide a mechanism to configure CDI spec folders if desired at a later stage.